examples/gnrc_border_router/README.md

```diff
@@ -217,7 +217,7 @@ USEMODULE += gnrc_slip
 ## Configuration
 
 In order to connect a RIOT 6LoWPAN border router over SLIP you run a small
-program called tunslip (imported from Contiki) [2] on the Linux host. The
+program called tunslip6 (imported from Contiki) [2] on the Linux host. The
```

Comment:
> The instructions for ethos are given at the beginning of this readme.

Arghs, sorry

> Should we give also instructions for an ethernet device?

Nope, people should know how to plug-in an ethernet cable and start uhcp ;-)